.markdownlint.jsonc

```
@@ -6,5 +6,6 @@
     },  // Set maximum line length to 88 characters, except in code blocks and tables
     "MD024": false,  // Allow duplicate headings
     "MD033": false,  // Allow inline HTML
-    "MD059": false  // Allow non-descriptive link text
+    "MD059": false,  // Allow non-descriptive link text
+    "MD040": false  // Allow fenced code blocks to not have a language specified
```

I find this to be quite nice though, and we can always just use ```text if it's plaintext.

```suggestion
    "MD059": false  // Allow non-descriptive link text
```